events-amd.js

```diff
@@ -9,7 +9,7 @@ define('event', [], function() {
 
 	function removeEvent(el, eventName, named) {
 		if (el['_event']) {
-			if (named) {
+			if (named && el['_event'][eventName]) {
```

Comment:
I've added a test on doc-amd to prevent this issue from happening. It didn't fail and, I figured later, this is because as soon as one adds a listener to some event, the second condition never returns false.